docs/api/autogluon.core.space.rst

```diff
@@ -11,7 +11,7 @@ Search space of possible hyperparameter values to consider.
    Define a dummy training function with searchable spaces for hyperparameters `lr` and `wd`:
 
    >>> import numpy as np
-   >>> import autogluon as ag
+   >>> import autogluon.core as ag
```

Comment:
suggest to also test these codes. one option is using [doctest](https://docs.python.org/3/library/doctest.html). not necessary to do it right now, please consider enable it in a future PR if not enabled before.

Reply:
It is tested through d2lbook + sphinx. Big part of this CR was making sure that the docs compile.